### PR TITLE
Network watchers

### PR DESCRIPTION
**Pull request by fdistorted**

### CODE REVIEW

**Review by irmatov — CHANGES_REQUESTED** [review]

File: resources/network_watchers.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	result, err := svc.ListAll(ctx)
+	if err != nil {
+		return err
+	}
+	for _, w := range *result.Value {
+		resourceDetails, err := client.ParseResourceID(*w.ID)
+		if err != nil {
+			return err
+		}
+		svc := meta.(*client.Client).Services().Network.Watchers
```
[the remark this answers is not in the document]
Comment (fdistorted) [reply resources/network_watchers.go:115]: it is also a duplicate of one that is already out of the loop.

[the remark this answers is not in the document]
Comment (fdistorted) [reply resources/network_watchers.go:115]: I am a bad copypaster ^^

**Review by irmatov — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]
LGTM, fix conflicts and we can merge

**Pull request merged**